packages/step-storybook/package.json

```diff
@@ -24,7 +24,9 @@
 		"typescript": "^4.0.3"
 	},
 	"peerDependencies": {
+		"@babel/core": ">= 7",
```

Comment:
I don't think we need @babel/core here - core-js should be sufficient

Reply:
on second thought- how does the require() chain look for core-js look? Because this package is built directly with tsc, core-js requires aren't going to be weaved into the lib code

Reply:
Keeping it